src/component/Input/index.ts

```diff
@@ -0,0 +1 @@
+export {Input} from './input';
```

Comment:
Имя компонента должно быть с большой буквы. Исправь само имя и имя в пути импорта
```suggestion
export {Input} from './Input';
```